configs/pare/hrnet_w32_conv_pare_mix.py

```diff
@@ -1,12 +1,16 @@
+_base_ = ['../_base_/default_runtime.py']
 use_adversarial_train = True
 
+# evaluate
 evaluation = dict(interval=6, metric=['pa-mpjpe', 'mpjpe'])
+
+img_res = 224
+
+# optimizer
 optimizer = dict(
-    backbone=dict(type='Adam', lr=5.0e-5),
-    head=dict(type='Adam', lr=5.0e-5),
-)
+    backbone=dict(type='Adam', lr=5.0e-5), head=dict(type='Adam', lr=5.0e-5), )
```

Comment:
This line is too long and might cause the lint error, the same with other configs.